cibuildwheel/windows.py

```diff
@@ -136,12 +131,27 @@ def shell(args, env=None, cwd=None):
 
         # run the before_build command
         if before_build:
-            before_build_prepared = prepare_command(before_build, project=abs_project_dir)
+            before_build_prepared = prepare_command(before_build, {'project': abs_project_dir})
             shell([before_build_prepared], env=env)
 
         # build the wheel
+        if os.path.exists(built_wheel_dir):
+            shutil.rmtree(built_wheel_dir)
+        os.makedirs(built_wheel_dir)
```

Comment:
Why was this moved?

Reply:
I prefer to have it close to the code that actually use the directory.